components/secret-mgt/org.wso2.carbon.identity.secret.mgt.core/src/main/java/org/wso2/carbon/identity/secret/mgt/core/SecretManager.java

```diff
@@ -81,8 +86,42 @@
     /**
      * This function is used to replace a given secret.
      *
-     * @param secret secret object.
+     * @param secretTypeName Name of the {@link SecretType}.
+     * @param secret         secret object.
```

Comment:
Fix formatting. -->`The Secret object.`

Reply:
Addressed with commit [001ee25](https://github.com/wso2/carbon-identity-framework/pull/3762/commits/001ee2501f3eab862efd07a13864e81939cfe648)